mmv1/products/migrationcenter/PreferenceSet.yaml

```diff
@@ -0,0 +1,197 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+base_url: projects/{{project}}/locations/{{location}}/preferenceSets
+create_url: projects/{{project}}/locations/{{location}}/preferenceSets?preferenceSetId={{preference_set_id}}
+self_link: projects/{{project}}/locations/{{location}}/preferenceSets/{{preference_set_id}}
+id_format: projects/{{project}}/locations/{{location}}/preferenceSets/{{preference_set_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/preferenceSets/{{preference_set_id}}
+name: PreferenceSet
+description: Manages the PreferenceSet resource.
+update_verb: :PATCH
+update_mask: true
+autogen_async: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'preference_set_basic'
+    primary_resource_id: 'default'
+    vars:
+      set_name: 'preference-set-test'
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    description: 'Output only. Name of the preference set. '
+    output: true
+  - !ruby/object:Api::Type::String
+    name: createTime
+    description: 'Output only. The timestamp when the preference set was created. '
+    output: true
+  - !ruby/object:Api::Type::String
+    name: updateTime
+    description: 'Output only. The timestamp when the preference set was last updated. '
+    output: true
+  - !ruby/object:Api::Type::String
+    name: displayName
+    description: 'User-friendly display name. Maximum length is 63 characters. '
+  - !ruby/object:Api::Type::String
+    name: description
+    description: 'A description of the preference set. '
+  - !ruby/object:Api::Type::NestedObject
+    properties:
+      - !ruby/object:Api::Type::String
+        name: targetProduct
```

Comment:
Interesting. It doesn't seem that the field is exposed in the API doc

Reply:
Seems like this is just an issue with the docs not being updated. They're both present in the discovery docs: https://migrationcenter.googleapis.com/$discovery/rest?version=v1